### DISCUSSION TITLE
Firefox console message

### DISCUSSION THREAD

**Discussion by SpinB** [Cookie-AutoDelete/Cookie-AutoDelete#1058] [Q&A / Support] [answered]
1614162119210	addons.webextension.CookieAutoDelete@kennydo.com	WARN	pluginData has been deprecated (along with Flash plugin support)

What functionality the extension lost??

**Comment by kennethtran93** [comment, marked as answer]
Nothing much, as pluginData refers to the flash plugin, and flash is more or less deprecated and the major browsers have already blocked flash from running anymore.

Plugin data essentially clears the flash plugin data, including flash cookies.

Chrome already ignores calls to that function, and I suspect Firefox will soon as well, given the warning prompt.

**Reply by SpinB** [reply]
I forgot to mention but this is in Firefox 86 and i was not seeing it before, is this because of the new Tracking Cookie Protection/Super Cookie?

**Reply by kennethtran93** [reply]
Nope, as Adobe flash is now considered dead anyway, so no flash applications should be able to run.  They probably included that warning in the latest versions of Firefox to warn developers that they will soon phase out this function.  The new tracking protection has very little, if any, to do with plugindata at all.
